tests/cases/standalone/common/alter/alter_table.result

```diff
@@ -78,14 +124,40 @@ DESC TABLE test_alt_table;
 | i      | Int32                | PRI | YES  |         | TAG           |
 | j      | TimestampMillisecond | PRI | NO   |         | TIMESTAMP     |
 | k      | String               | PRI | YES  |         | TAG           |
-| m      | Int32                |     | YES  |         | FIELD         |
+| m      | Boolean              |     | YES  |         | FIELD         |
 | dt     | TimestampMicrosecond |     | YES  |         | FIELD         |
 +--------+----------------------+-----+------+---------+---------------+
 
 DROP TABLE test_alt_table;
 
 Affected Rows: 0
 
+-- test if column with default value can change type properly
+CREATE TABLE test_alt_table_default(h INTEGER, i INTEGER DEFAULT 0, j TIMESTAMP TIME INDEX, PRIMARY KEY (h));
```

Comment:
non-blocking: Can we add a case that the default value is an integer value, and we alter the column to string type?